index.js

```diff
@@ -0,0 +1,234 @@
+        + ' and in chamber music from the Academy of Santa Cecilia with'
+        + ' Riccardo Brengola.',
+  },
+  {
+    performerPicture: './assets/mobile/performers/federico-piccotti.jpg',
+    performername: 'Federico Piccotti',
+    alternativeText: 'Federico Piccotti',
+    post: 'Violino',
+    bar: '',
+    performerDescription: 'Born in Rome, he graduated from the Conservatory of santa'
+        + 'Cecilia and subsequently specialised at the National Academy of Santa Cecilia.',
+  },
+  {
+    performerPicture: './assets/mobile/performers/tomasz-sierant.gif',
+    performername: 'Tomasz Sierant',
+    alternativeText: 'Tomasz Sierant',
+    post: 'Flaute',
+    bar: '',
+    performerDescription: 'Polish flutist Tomasz Sierant: born in 1995, began music study 2008;'
+        + 'immediately became successful in flute competitions; received music education in Poland, Belgium and Switzerland',
+  },
+];
+
+const hiddenPerformers = document.createElement('div');
+hiddenPerformers.classList.add('hidden-performers');
+
+for (let i = 0; i < performersInfo.length; i += 1) {
+  const performersContainer = document.createElement('div');
+  performersContainer.classList.add(
+    'performer-container',
+    'py-2',
+    'd-flex',
+    'justify-content-around',
+    'gap-5',
+  );
+  const performerLeft = document.createElement('div');
+  performerLeft.classList.add(
+    'performer-bg',
+    'me-3',
+    'me-md-5',
+    'mt-3',
+  );
+
+  const performerImg = document.createElement('img');
+  performerImg.classList.add('performer-img');
+  performerImg.src = performersInfo[i].performerPicture;
+  performerImg.alt = performersInfo[i].alternativeText;
+  performerImg.width = 130;
+  performerImg.height = 100;
+
+  performerLeft.appendChild(performerImg);
+
+  const performerRight = document.createElement('div');
+  performerRight.classList.add(
+    'performer-right',
+    'bg-light',
+    'p-3',
+  );
+
+  const performerName = document.createElement('h3');
+  performerName.classList.add('performer-name');
+  performerName.innerHTML = performersInfo[i].performername;
+
+  const performerPost = document.createElement('p');
+  performerPost.classList.add(
+    'performer-post',
+  );
+
+  performerPost.innerHTML = performersInfo[i].post;
+
+  const grayBar = document.createElement('span');
+  grayBar.classList.add('gray-bar');
+
+  const performerPara = document.createElement('p');
+  performerPara.classList.add('performer-para');
+  performerPara.innerHTML = performersInfo[i].performerDescription;
+
+  performerRight.append(
+    performerName,
+    performerPost,
+    grayBar,
+    performerPara,
+  );
+
+  performersContainer.append(
+    performerLeft,
+    performerRight,
+  );
+  if (i < 2) {
+    performersSection.appendChild(performersContainer);
+  } else {
+    performersSection.appendChild(performersContainer);
+    performersContainer.classList.add('d-none');
+  }
+}
+
+const showmore = document.createElement('div');
```

Comment:
- [ ] Kindly make sure on the Desktop version of the page we don't have the see-more feature. only the mobile version has this feature as it is in the provided template.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203992724-487d5e63-4640-4212-b1ae-e434c8b3aa85.png)

Template design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203992835-b276d2e1-9e13-46ce-8052-4444f722dbfd.png)